src/app/AddCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { useContext, useState } from 'react'
+import { CardInfoContext, UpdateCardInfoContext } from '../context/paymentContext'
+import { CardBox, CardCvc, CardName, CardNumber, CardPassword, CardType, CardDate } from '../components'
+import { UpdateFunnelStepContext } from '../context/funnelStepContext'
+import ui from '../styles/index.module.css'
+
+const AddCard = () => {
+  const cardInfo = useContext(CardInfoContext)
+  const updateCardInfo = useContext(UpdateCardInfoContext)
+  const updateFunnelStep = useContext(UpdateFunnelStepContext)
+
+  const [isShowModal, setIsShowModal] = useState(false)
+
+  return (
+    <main>
+      <h2>1️⃣ 카드 추가</h2>
+      <div className={ui['root']}>
+        <div className={ui['app']}>
+          <h2 className={ui['page-title']}>
+            <button onClick={() => updateFunnelStep({ step: 'list' })}>&lt;</button>
+            &nbsp; 카드 추가
+          </h2>
+          <CardBox />
+          <CardNumber />
+          <CardDate />
+          <CardName />
+          <CardCvc />
+          <CardPassword />
+          <div className={ui['button-box']}>
+            <button
+              onClick={() => {
+                if (cardInfo.cardType) {
+                  updateCardInfo({ ...cardInfo, cardNo: cardInfo.cardList?.length || 0 })
+                  updateFunnelStep({ step: 'complete' })
+                } else setIsShowModal((state) => !state)
+              }}
```

Comment:
```suggestion
                if (cardInfo.cardType) {
                  updateCardInfo({ ...cardInfo, cardNo: cardInfo.cardList?.length || 0 })
                  updateFunnelStep({ step: 'complete' })
                  return
                }
                setIsShowModal((state) => !state)
              }}
```

else를 써도 무방합니다. 다만 else없이 ealry return을 통해 분리할 수도 있어요